cypress/support/pages/tasks/pageTasks.js

```diff
@@ -80,6 +80,51 @@ class Tasks {
 	static #downloadFileButton = '[data-testid="file-download-btn-0"]';
 	static #fileRenameButton = '[data-testid="file-rename-btn-0"]';
 	static #fileDeleteButton = '[data-testid="file-delete-btn-0"]';
+	static #importModalTaskNameInput = '[data-testid="import-modal-name-input"]';
+	static #taskCardTitleCourseDetail = '[data-testid="task-card-title-0"]';
+	static #taskPublishButtonOnTaskCardCourseDetail =
+		'[data-testid="task-card-action-publish-0"]';
+	static #submissionTabOnTaskDetail = '[id="submission-tab-link"]';
+	static #fileTitleInTaskDetail = '[data-testid="file-title-card-0"]';
+
+	verifyAttachedFilesInTaskDetail() {
+		cy.get(Tasks.#filesSection).should("be.visible");
+		cy.get(Tasks.#fileTitleInTaskDetail).should("be.visible");
+	}
+
+	verifyTaskDetailPage() {
+		cy.get(Tasks.#detailsTab).should("be.visible");
+		cy.get(Tasks.#submissionTabOnTaskDetail).should("be.visible");
+	}
+
+	clickDraftTaskCardInCourseDetail() {
+		cy.get(Tasks.#taskCardTitleCourseDetail).click();
+	}
+
+	checkTaskPublishButtonVisibleOnCourseDetail() {
+		cy.get(Tasks.#taskPublishButtonOnTaskCardCourseDetail).should("be.visible");
+	}
+
+	checkTaskIsDraftInCourseDetail() {
+		const draftWords = ["Draft", "Entwurf", "Borrador", "Чернетка"];
```

Comment:
- since we set language initially to Deutsch then we don't need this array
- the method itself can be much simpler and not require so much effort

```suggestion
		const draftWord = "Entwurf";
		cy.get(Tasks.#taskCardTitleCourseDetail)
			.should("be.visible")
			.invoke("text")
			.should("include", draftWord);
```